packages/language-server/src/utils.ts

```diff
@@ -38,20 +38,16 @@ export function getLastPartOfPath(path: string): string {
 }
 
 /**
- *
- * The language service is case insensitive, and would provide
- * hover info for Svelte components like `Option` which have
- * the same name like a html tag.
+ * Return true if a specific node could be a component.
+ * This is not a 100% sure test as it'll return false for any component that does not match the standard format for a component
  */
 export function isPossibleComponent(node: Node): boolean {
-	return !!node.tag?.[0].match(/[A-Z]/);
+	return !!node.tag?.[0].match(/[A-Z]/) || !!node.tag?.match(/.+[.][A-Z]/);
```

Comment:
This was added to match imported component under a namespace ex: 

```astro
---
import * as thing from "component.tsx"
---

<thing.Counter/>
```

Reply:
Nice catch!